Classes/Client/Event/ModifyServiceBwResponseEvent.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package jweiland/service_bw2.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace JWeiland\ServiceBw2\Client\Event;
+
+/**
+ * An event to modify the response of a Service BW API request.
+ *
+ * The event will be fired before the response gets cached and before paginated requests get merged.
+ */
+final class ModifyServiceBwResponseEvent
+{
+    private $path;
```

Comment:
Please add data type as long as we do not have PHP 7.4 by default. And add a default value. I know, it will be set immediately by __contructor, but there a mechanism in TYPO3 and Doctrine which prevents calling the constructor.